src/app/qgsattributesformproperties.cpp

```diff
@@ -1046,6 +1050,17 @@ void DnDTree::onItemDoubleClicked( QTreeWidgetItem *item, int column )
         layout->addRow( showAsGroupBox );
       }
 
+      QgsCollapsibleGroupBox *styleGroupBox = new QgsCollapsibleGroupBox( tr( "Style" ), layout->widget() );
+      styleGroupBox->setObjectName( QLatin1Literal( "attributeFormPropertiesContainerStyle" ) );
+      QFormLayout *customizeGroupBoxLayout = new QFormLayout( styleGroupBox ) ;
+      QgsColorButton *backgroundColorButton = new QgsColorButton( styleGroupBox, tr( "Select the background color for the container" ) );
```

Comment:
I think the title here should just be "Container Background Color" for consistency with other color dialogs